solo-io/gloo
# *: Add initial support for the GW API 1.1 release

# Description



This commit introduces initial support for the upstream GW API 1.1 release. The 1.1 release had several changes that we need to evaluate and implement support for during the 1.18 cycle. What's relevant here is that the conformance suite has been overhauled in the 1.1 suite, so this commit adopts the new suite implementation.

## Testing steps



1. Run the standard suite: `make conformance`.
2. Run the extended suite: `make conformance-extended`
3. Verify the generated report matches the new 1.1 formatting: `cat _test/conformance/<version>-report.yaml`

## Notes for reviewers



I pinned the controller-runtime dependency to match the same k8s.io that are being pinned to the 0.29.x release. This was primarily done as builds were breaking with 0.18.x c-r versions. We can tackle removing this replace pin when we update Gloo with 0.30.x k8s.io dependencies. 

# Checklist:
